project/src/common.js

```diff
@@ -7,4 +7,16 @@ const getStarRating = (rating) => {
   return offerRatingInPercents;
 };
 
-export { getStarRating };
+const getDate = function (comment) {
+  const date = new Date(comment.date);
+  const year = date.getUTCFullYear().toString();
+  const month = date.toLocaleString('en', { month: 'long' });
+
+  return {
+    date,
+    year,
+    month,
+  };
```

Comment:
```suggestion
const getFormattedDate = function (date) {
  const date = new Date(date);
  const year = date.getUTCFullYear().toString();
  const month = date.toLocaleString('en', { month: 'long' });

  // верни отформатированную строку. Мы это уже обсуждали
  return ''
```
Этот хелпер НИЧЕГО не должен знать об комменте и тд. Ему приходит дата - он её форматирует. Именно в таком хелпера как сейчас смысла ноль